src/ServerlessBuildPlugin.ts

```diff
@@ -239,12 +240,43 @@ export class ServerlessBuildPlugin {
     }
   }
 
+  async buildFunction (fnName, fnConfig) {
+
+    const runtime = (fnConfig.runtime && fnConfig.runtime) || this.serverless.config.service.runtime;
+
+    this.logger.message('FUNCTION', c.reset.bold(fnName));
+    this.logger.log('');
+
+    if (runtime.indexOf('nodejs') > -1) {
+      return this.buildNodejsFunction(fnName, fnConfig);
+    }
+    if (runtime.indexOf('java') > -1) {
+      return this.buildJavaFunction(fnName, fnConfig);
+    }
+
+    throw new this.serverless.classes.Error(
+      '[serverless-build-plugin]: Unsupported runtime. Only supports nodejs and java.',
+    );
+  }
+
+  async buildJavaFunction (fnName, fnConfig) {
```

Comment:
Considering this does nothing, you should call this `passthroughFunction`

Reply:
Yep sure.